zerver/views/webhooks/taiga.py

```diff
@@ -0,0 +1,231 @@
+            'renamed': u':notebook: %(user)s renamed sprint from %(old)s to **%(new)s**.',
+            'estimated_start': u':calendar: %(user)s changed estimated start of sprint **%(subject)s** from %(old)s to %(new)s.',
+            'estimated_finish': u':calendar: %(user)s changed estimated finish of sprint **%(subject)s** from %(old)s to %(new)s.',
+            'delete': u':x: %(user)s deleted sprint **%(subject)s**.'
+        },
+        'task': {
+            'create': u':clipboard: %(user)s created task **%(subject)s**.',
+            'set_assigned_to': u':busts_in_silhouette: %(user)s assigned task **%(subject)s** to %(new)s.',
+            'changed_assigned_to': u':busts_in_silhouette: %(user)s reassigned task **%(subject)s** from %(old)s to %(new)s.',
+            'blocked': u':lock: %(user)s blocked task **%(subject)s**.',
+            'unblocked': u':unlock: %(user)s unblocked task **%(subject)s**.',
+            'set_milestone': u':calendar: %(user)s added task **%(subject)s** to sprint %(new)s.',
+            'changed_milestone': u':calendar: %(user)s changed sprint of task **%(subject)s** from %(old)s to %(new)s.',
+            'changed_status': u':chart_with_upwards_trend: %(user)s changed status of task **%(subject)s** from %(old)s to %(new)s.',
+            'renamed': u':notebook: %(user)s renamed task %(old)s to **%(new)s**.',
+            'description': u':notebook: %(user)s updated description of task **%(subject)s**.',
+            'commented': u':thought_balloon: %(user)s commented on task **%(subject)s**.',
+            'delete': u':x: %(user)s deleted task **%(subject)s**.',
+            'changed_us': u':clipboard: %(user)s moved task **%(subject)s** from user story %(old)s to %(new)s.'
+        },
+        'issue': {
+            'create': u':bulb: %(user)s created issue **%(subject)s**.',
+            'set_assigned_to': u':busts_in_silhouette: %(user)s assigned issue **%(subject)s** to %(new)s.', #
+            'changed_assigned_to': u':busts_in_silhouette: %(user)s reassigned issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_priority': u':rocket: %(user)s changed priority of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_severity': u':warning: %(user)s changed severity of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_status': u':chart_with_upwards_trend: %(user)s changed status of issue **%(subject)s** from %(old)s to %(new)s.',
+            'changed_type': u':bulb: %(user)s changed type of issue **%(subject)s** from %(old)s to %(new)s.',
+            'renamed': u':notebook: %(user)s renamed issue %(old)s to **%(new)s**.',
+            'description': u':notebook: %(user)s updated description of issue **%(subject)s**.',
+            'commented': u':thought_balloon: %(user)s commented on issue **%(subject)s**.',
+            'delete': u':x: %(user)s deleted issue **%(subject)s**.'
+        },
+    }
```

Comment:
I added the doc string explaining my concept of displaying new values in bold as well as emojis.